quill-cassandra-zio/src/main/scala/io/getquill/cassandrazio/Quill.scala

```diff
@@ -19,14 +19,12 @@ object Quill {
       ZLayer.fromFunction((session: CassandraZioSession) => new Cassandra[N](naming, session))
   }
 
-  case class Cassandra[+N <: NamingStrategy](val naming: N, session: CassandraZioSession)
+  class Cassandra[+N <: NamingStrategy](val naming: N, session: CassandraZioSession)
```

Comment:
Removed the `case` as all the other context are implemented as simple classes, not case classes. Doesn't seem to be necessary here anyway